src/style.scss

```diff
@@ -17,6 +17,9 @@
 // Components
 // ==========================================================================
 @import "vl-ui-info-tile/src/scss/info-tile";
+@import "vl-ui-link/src/scss/link";
+@import "vl-ui-toggle/src/scss/toggle";
+@import "vl-ui-accordion/src/scss/accordion";
```

Comment:
Is dit den import van den onderliggende accordion of van de vl-accordion?
Indien het 2e moet die nog weg denk ik?